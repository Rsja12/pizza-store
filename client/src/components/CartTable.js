import React from 'react'
import Table from 'react-bootstrap/Table'

import CartList from './CartList'

const CartTable = ({ cart }) => {
    return (
        <Table striped borderless>
            <thead>
                <tr>
                    <th>Pizza</th>
                    <th>Quantity</th>
                    <th className='d-flex justify-content-end'>Remove</th>
                </tr>
            </thead>
            <tbody>
                {
                    cart.map(pizza => (
                        <CartList pizza={pizza} key={pizza.id} />
                    ))
                }
            </tbody>
        </Table>
    )
}

export default CartTable
