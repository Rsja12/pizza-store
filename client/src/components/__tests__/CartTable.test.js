import React from 'react';
import { shallow } from 'enzyme';

import CartTable from '../CartTable';

it('renders CartTable component', () => {
    const mockCart = [
        {
            id: 1,
            pizza: 'pepperoni',
            quantity: 2,
        },
    ];

    expect(shallow(<CartTable cart={mockCart} />)).toMatchSnapshot();
});
