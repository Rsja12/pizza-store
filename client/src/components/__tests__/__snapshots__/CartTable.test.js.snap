// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CartTable component 1`] = `ShallowWrapper {}`;
